librz/core/cbin.c

```diff
@@ -2843,7 +2843,12 @@ RZ_API bool rz_core_file_info_print(RzCore *core, RzBinFile *binfile, RzCmdState
 		break;
 	case RZ_OUTPUT_MODE_JSON:
 		pj_o(state->d.pj);
-		pj_ks(state->d.pj, "file", filename);
+		const char *file_tag = "file";
+		if (rz_str_is_utf8(filename)) {
+			pj_ks(state->d.pj, file_tag, filename);
+		} else {
+			pj_kr(state->d.pj, file_tag, (unsigned char *)filename, strlen(filename));
```

Comment:
i disagree on this. this should be encoded properly by pj_ks using `\uxxxx` codes.
https://tools.ietf.org/id/draft-ietf-json-rfc4627bis-09.html#rfc.section.7